README.md

```diff
@@ -110,6 +110,14 @@ else
 fi
 ```
 
+#### Automatically starting and entering a user shell
+
+```ps1
+PS> wsl.exe -- sh -c "subsystemctl is-running || subsystemctl start ; subsystemctl shell --uid=1000"
```

Comment:
We have `shell --start` https://github.com/sorah/subsystemctl/blob/0e3cb762f7f5666ce08fbfe6cadac399be090443/src/main.rs#L40

Reply:
Whoops...